api/src/main/java/com.epam.demo/App.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.demo;
```

Comment:
make sure your package contains flow of folders `com/epam/demo` but not a dot-delimited folder